server/src/api/command-normalizer.ts

```diff
@@ -0,0 +1,18 @@
+import { join } from 'path';
+
+export function normalize(cwd: string, cmds: string[]): string[] {
```

Comment:
Some commands like `ng test ...` uses karma, which may have configurations that won't generate the expected output. This ensures that we can get what we need in the interim until build events.

Reply:
One of the design goals we follow is:
Running a command using Console should work the same way as running a command without console. Hence we cannot override karma configuration. The tests may not work or different tests may run.

The issue is actually broader:
There is no guarantee that  `ng test` will actually use karma. It can also use `jest` or whatever the user provisions in angular.json.

Given that, do you think it makes sense to get the builder name from angular.json to make sure `ng test` even uses karma before we start parsing the output?